examples/benchmark/README.md

```diff
@@ -0,0 +1,91 @@
+# Performance evaluation
+
+With the help of this guide you are able to take performance measurments in your setup using STUNner. Both running STUNner locally (outside of Kubernetes) and running STUNner in Kubernetes can be evaluated.
+
+Compare the locally measured results to the result measured in Kubernetes and figure out the overhead cost. The extra cost of your cluster's networking may surprise you in terms of extra delay or more packet drops using the same bandwidth.
+
```

Comment:
If you could give a rough time estimate (for install and measure) that would be great extra.